.gitignore

```diff
@@ -3,6 +3,7 @@
 
 *.pyc
 *.egg-info
+docs/_build/
```

Comment:
Don't put this inline, use the tox temporary folders instead 👍

Reply:
After building the docs, they need to stay around so that they can be inspected (e.g.: with Firefox). I'm not sure a temporary directory would work, since that'd be cleaned right away.

Maybe I should have been clearer, this serves two purposes:

- Testing it in CI.
- Building and inspecting locally.

Reply:
Temporary only means it's deleted at startup of a new doc build run, not that after run. You only need one, just move it into the tox to be out of way.

Reply:
Aaah, now I get what you meant. Makes perfect sense.

Reply:
Check out how virtualenv does things, I think that worked well in the past and is not too complicated 👍

Reply:
Yup, that's exactly why I did. Forgot to push :sweat_smile: